lib/datadog/appsec/contrib/sinatra/patcher.rb

```diff
@@ -11,6 +11,7 @@
 require_relative 'gateway/route_params'
 require_relative 'gateway/request'
 require_relative '../../../tracing/contrib/sinatra/framework'
+require_relative '../../../tracing/contrib'
```

Comment:
That's a good catch!

But `AppSec` should be able to load without `Tracing` being present, so I think instead there's a bit of conditioning around the `Tracing` constants to be made below.